### PR TITLE
Fix minor issues in lip 0055

### PR DESCRIPTION
**Pull request by AndreasKendziorra**
The following is done:
- fixing outdated links
- mentioning that some constants are configurable
- add processing step **Command verification**
- improve description of **Transaction verification** step
- mention block schema validation in *block validation* section
- fixing more minor issues like broken grammar

### CODE REVIEW

File: proposals/lip-0055.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ This LIP introduces the following new block header properties:
 - `stateRoot`: The root of the sparse Merkle tree that is computed from the state of the blockchain. See the [Define state model and state root][lip-0040] LIP for the reason why it needs to be included in a block header.
 - `assetsRoot`: The root of the Merkle tree computed from the block assets array. See [below](#separation-between-block-header-and-block-assets) for more details.
 - `generatorAddress`: The address of the block generator. It replaces the `generatorPublicKey` property. See [below](#change-generator-public-key-to-generator-address) for more details.
-- `aggregateCommit`: This property contains the aggregate signature for a certificate for a certain block height. Based on this, any node can create a certificate for the corresponding height. See the [Introduce certificate generation mechanism][research:certificate-generation] LIP for more details.
+- `aggregateCommit`: This property contains the aggregate signature for a certificate for a certain block height. Based on this, any node can create a certificate for the corresponding height. See the [LIP 0061][lip-0061] for more details.
```
Comment (ricott1) [inline proposals/lip-0055.md:45]:
```suggestion
- `aggregateCommit`: This property contains the aggregate signature for a certificate for a certain block height. Based on this, any node can create a certificate for the corresponding height. See [LIP 0061][lip-0061] for more details.
```

**Review by ricott1 — APPROVED** [review]
A minor typo

**Comment by ricott1** [comment]
As you mention in the tests, we could also reconsider the order of fork choice and static validation as part of this PR

**Review by MaximeGagnebin — APPROVED** [review]
The changes look good. 
I would suggest to update all the LIP links if we are already doing it for some of them (certificate generation -> LIP 61). I spotted the following that could be updated
- Define state model and state root -> LIP 40
- Introduce BFT module -> LIP 58
- Update genesis block schema and processing -> LIP 60

File: proposals/lip-0055.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ Furthermore, in the following we indicate with `block` be the block under consid
 
 *Figure 1: A schematic depiction of the various stages of a block processing. The steps performed in the consensus domain are indicated by blue boxes, while the steps performed in the state-machine domain are indicated by red boxes. Within the state-machine domain, the transactions processing is indicated by purple boxes. The network domain is indicated by green boxes.*
 
-In this section, we describe the various processing stages of a block. Note that a genesis block follows different rules specified in the ["Update genesis block schema and processing" LIP][research:update-genesis-block].
+In this section, we describe the various processing stages of a block. Note that a genesis block follows different rules specified in ["LIP 0060][lip-0060].
```
Comment (MaximeGagnebin) [inline proposals/lip-0055.md:91]:
```suggestion
In this section, we describe the various processing stages of a block. Note that a genesis block follows different rules specified in [LIP 0060][lip-0060].
```

File: proposals/lip-0055.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ The second change is to update the block header schema. In general, it is desira
 - does not need to be changed when modules are added or removed to a running blockchain,
 - is used by every blockchain in the Lisk ecosystem regardless of the modules implemented in the individual chains.
 
-Furthermore, we update the block header schema to include new properties introduced by the ["Define state model and state root"][lip-0040] LIP, the ["Introduce BFT module"][research:bft-module] LIP, and the ["Introduce certificate generation mechanism"][research:certificate-generation] LIP.
+Furthermore, we update the block header schema to include new properties introduced by ["LIP 0040"][lip-0040], ["LIP 0058"][lip-0058], and [LIP 0061][lip-0061].
```
Comment (MaximeGagnebin) [inline proposals/lip-0055.md:32]:
```suggestion
Furthermore, we update the block header schema to include new properties introduced by [LIP 0040][lip-0040], [LIP 0058][lip-0058], and [LIP 0061][lip-0061].
```

**Review by MaximeGagnebin — APPROVED** [review]
Please consider the two small formatting suggestion.

**Review by shuse2 — APPROVED** [review]

**Review by Incede — APPROVED** [review]

**Review by ricott1 — APPROVED** [review]

**Review by karmacoma — APPROVED** [review]

**Pull request merged**